spec.html

```diff
@@ -21438,6 +21730,21 @@ <h1>Runtime Semantics: ClassFieldDefinitionEvaluation</h1>
       </emu-note>
     </emu-clause>
 
+    <emu-clause id="sec-runtime-semantics-classstaticblockdefinitionevaluation" type="sdo" aoid="ClassStaticBlockDefinitionEvaluation">
+      <h1>Runtime Semantics: ClassStaticBlockDefinitionEvaluation</h1>
+      <p>With parameter _homeObject_.</p>
+      <emu-grammar>ClassStaticBlock: `static` `{` ClassStaticBlockBody `}`</emu-grammar>
+      <emu-alg>
+        1. Let _lex_ be the running execution context's LexicalEnvironment.
+        1. Let _privateScope_ be the running execution context's PrivateEnvironment.
+        1. Let _sourceText_ be the empty sequence of Unicode code points.
+        1. Let _body_ be OrdinaryFunctionCreate(%Function.prototype%, _sourceText_, &laquo; &raquo;, |ClassStaticBlockBody|, ~non-lexical-this~, _lex_, _privateScope_).
+        1. Perform MakeMethod(_body_, _homeObject_).
+        1. Return the ClassStaticBlockDefinition Record { [[Body]]: _body_ }.
+      </emu-alg>
+      <emu-note>The function created for _body_ is never directly accessible to ECMAScript code.</emu-note>
```

Comment:
```suggestion
      <emu-note>The function _body_ is never directly accessible to ECMAScript code.</emu-note>
```

Reply:
This text mirrors the text for _ClassFieldDefinitionEvaluation_ just above it.

Reply:
Ah, yeah, that's wrong too. There isn't a function *created for* `_initializer_`/`_body_`; `_initializer_`/`_body_` *is* the function (that has been created for _Initializer_/_ClassStaticBlockBody_, if you like).